File: Java/SETPractice/src/main/java/SOLIDPrinciple/InterfaceSegregation/Lead.java
package SOLIDPrinciple.InterfaceSegregation;

public class Lead implements ILead,IProgrammer{
    @Override
    public void CreateTask() {
        System.out.println("Lead created task");
    }

    @Override
    public void AssignTask() {
        System.out.println("Lead assigned task");
    }

    @Override
    public void WorkOnTask() {
        System.out.println("Lead worked on the task");
    }
}

File: DotNet/SETPractice/SOLIDPrinciples/InterfaceSegregation/Lead.cs
namespace SOLIDPrinciples.InterfaceSegregation
{
    public class Lead : ILead, IProgrammer
    {
        public void AssignTask()
        {
            Console.WriteLine("Lead assign task");
        }

        public void CreateTask()
        {
            Console.WriteLine("Lead create task");
        }

        public void WorkOnTask()
        {
            Console.WriteLine("Lead working on task");
        }
    }
}
